File: OOP_Ponavljanje/php/prviObjekt.php
<?php 

//definicija klase Student
class Student {
//podatkovni clanovi
	public $ime;
	private $matBr;

//konstruktor
	public function __construct($ime, $matBr) {
		$this->ime = $ime;
		$this->matBr = $matBr;
	}

	public function __get($name) {
		echo $name;
	}

//javna metoda
	public function getMatBr() {
		return $this->matBr;
	}

	public function __destruct() {}
}

$pero = new Student("Petar", 25);
// $pero = "String bla bla";
// echo $pero->getMatBr();

echo $pero->matBr;
$ivana = new Student("Ivana", 123123);
// $mate = new Student("Mate", 112233);

// echo $pero->getMatBr();

File: OOP_Ponavljanje/c++/prviObjekt.cpp
#include<iostream>
using namespace std;

class Student {
	public:
		string ime;
	private:
		int matBr;

	public:
	Student(string ime, int matBr) {
		this->ime = ime;
		this->matBr = matBr;
	}

	int getMatBr() {
		return this->matBr;
	}

};

int main() {
	Student* pero = new Student("Mate", 25);
	cout << pero->getMatBr() << endl;
	return 0;	
}